cmake/ExternalLibCEED.cmake

```diff
@@ -0,0 +1,104 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
`LibCEED` rather than `LIBCEED`? Guessing this inconsistency with the other `LIB` is CEED's doing.